src/modules/tools/spindle/SoftSerial/SoftSerial_tx.cpp

```diff
@@ -0,0 +1,87 @@
+/* 
+ * SoftSerial by Erik Olieman
+ * Date: 05 Jul 2014
+ * Revision: 10:236fce2e5b8c
+ * URL: http://developer.mbed.org/users/Sissors/code/SoftSerial/
+ */
+
+#include "SoftSerial.h"
+
+int SoftSerial::_putc(int c)
+{
+    while(!writeable());
+    prepare_tx(c);
+    tx_bit = 0;
+    txticker.prime();
+    tx_handler();
+    return 0;
+}
+
+void SoftSerial::send_break(void) {
+    while(!writeable());
+    tx_bit = 0;         //Just to make sure it appears as non-writable to other threads/IRQs
+    tx->write(0);
+    wait_us((bit_period * _total_bits * 3) / 2);

```

Comment:
also not acceptable if it is more than a few us. even then it will destabilize the entire system.